homework/3_first_interactive_component.md

```diff
@@ -1,4 +1,4 @@
-# Homework &mdash; First interactive component
+# Homework 3 -- First interactive component
```

Comment:
What is the reason to use -- ?

Reply:
consistent across all the assignments

Reply:
no special reason I just edited the titles to all look the same.

Reply:
Ok, I don't like those minuses and I would prefer to use either &mdash; or &ndash; everywhere. But I can leave with it ;-)